### ISSUE TITLE
JSON.stringify not working correctly on arrays when function is called from HTTP request

### ISSUE THREAD

**Issue by joaobarcia** [orientechnologies/orientdb#5213]
Hi,

Given javascript function with parameter "content"
```
return JSON.stringify(content);
```

And you have example content:
```
[{"type":"unitSection","subContent":[{"type":"youtube","youtubeVidID":"jNESnky3RVg"}]},{"type":"unitSection","subContent":[{"type":"text","text":"yo"}]},{"type":"unitEvaluationSection","evaluationType":"userConfirmation"}]
```
(this is our specific case, but even a simple [1,2,3] array doesn't work)

If you run the function through the studio, it returns the supposed value. If you run the function through http request you receive a blank response.

**Comment by joaobarcia** [comment]
Related question:

We spent a lot of time until we figured this is where our function was breaking. Our specific use case is that we want to send an object through the body of an http request to be saved as an embeddedlist parameter of a vertex. 

On the orient side what we do is: 
- receive the object
- pass it through JSON.stringify
- add it to an update query string
- running db.command

Is the the correct/best procedure?
